### ISSUE TITLE
undo redo

### ISSUE THREAD

**Issue by no-1ne** [alyssaxuu/screenity#85]
Hello Alyssa,
As many mentioned already Thank you for sharing your work. Its amazing..

Coming to the issue,
It would be great if you can add undo redo in fabricjs
this may help https://github.com/tbolis/react-sketch/blob/master/src/history.js

**Comment by alyssaxuu** [comment]
Yeah, undo/redo would be easy to implement, I already have my own library I've used in other fabric.js projects of mine :) The issue is more UX-wise, I don't really want to add more buttons in there, plus if I used keyboard shortcuts this would be problematic in several websites, e.g. if you want to annotate on top of Figma for example, undoing/redoing would apply to both Screenity and Figma. So it's something that might need some thinking.
